app/src/main/java/com/framgia/wsm/data/source/RequestRepository.java

```diff
@@ -169,7 +169,7 @@ public RequestRepository(RequestRemoteDataSource remoteDataSource) {
         requestOverTime3.setCreatedAt("23/06/2017");
         requestOverTime3.setFromTime("21/06/2017 18:00");
         requestOverTime3.setToTime("22/06/2017 19:00");
-        requestOverTime3.setStatus(2);
+        requestOverTime3.setStatus("rejected");
```

Comment:
ERROR: [WrongConstant] Must be one of: StatusCode.ACCEPT_CODE, StatusCode.PENDING_CODE, StatusCode.REJECT_CODE
 Ensures that when parameter in a method only allows a specific set of constants, calls obey those rules.